packages/block-library/src/embed/test/__snapshots__/index.js.snap

```diff
@@ -14,10 +14,10 @@ exports[`core/embed block edit matches snapshot 1`] = `
         aria-hidden="true"
         class="dashicon dashicons-embed-generic"
         focusable="false"
-        height="20"
+        height="24"
```

Comment:
I think this is an issue actually. As outlined here https://github.com/WordPress/gutenberg/pull/10651#discussion_r227423954 Dashicons should be 20x20 by default.

Reply:
Oh, that's a fair point. I'll reflect this in the component then, and will add some tests too.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/10651/commits/c7959cfc3308034bfa1763cc35317f065cfbcb7e, thanks for the suggestion!